src/backend/marsha/core/models/video.py

```diff
@@ -46,6 +46,12 @@ class Video(BaseFile):
         blank=True,
     )
 
+    is_public = models.BooleanField(
```

Comment:
Doing so means that we could fetch publicly a timed_text_track for example. But this is wrong. The timed textex track belongs to a video, you can access it only throught the video object

Reply:
I didn't get why this would make the timed_text_track available. when I search for "BaseFile" I see that it is only used as base class for Document and Video, so adding the `is_public` field on it would do the same?

Reply:
You are right, I thought it was the base class used by all the resources.